### PR TITLE
Update a few descriptions for new protobuf settings

### PR DESCRIPTION
**Pull request by caveman99**

### CODE REVIEW

**Comment by jfirwin** [comment]
@sachaw I haven't taken a look at the trunk requirements here. What's needed to get this passing?

File: docs/blocks/_lora-regions.mdx [hunk 1]
```diff
@@ -12,6 +12,8 @@
 |    `RU`     |         Russia          |
 |    `IN`     |          India          |
 |  `NZ_865`   |   New Zealand 865MHz    |
+|  `UA_433`   |     Ukraine 433MHz      |
+|  `UA_868`   |     Ukraine 868MHz      |
```
Comment (jfirwin) [inline docs/blocks/_lora-regions.mdx:16]: 🇺🇦 🇺🇦 🇺🇦

**Review by jfirwin — CHANGES_REQUESTED** [review]
Looks great @caveman99, looks like it's asking to run the following:

`Incorrect formatting, autoformat by running 'trunk fmt'`

Once that's passing we can get this merged in.

**Comment by caveman99** [comment]
@jfirwin checks are all green now :-)

**Pull request merged**